benchmark/profile_restful_api.py

```diff
@@ -66,21 +66,28 @@ def __init__(self,
                  temperature: float = 0.8,
                  top_p: float = 1.0,
                  csv: str = '',
+                 api_key: Optional[str] = None,
+                 model_name: Optional[str] = None,
                  **kwargs):
         self.tokenizer = Tokenizer(tokenzier_path)
         self.server_addr = server_addr
         self.temperature = temperature
         self.top_p = top_p
         self.csv = csv
-        client = APIClient(self.server_addr)
-        self.model_name = client.available_models[0]
+        self.api_key = api_key
+        client = APIClient(self.server_addr, api_key=self.api_key)
+        if model_name is None:
+            self.model_name = client.available_models[0]
+            print(f"using model:{self.model_name}")
```

Comment:
A small change, it is best for the print format to be consistent with the existing form in the script.

```python3
print(f'using model: {self.model_name}\n')
```

Reply:
Ok.